modules/devices/mediaDeviceHelper.js

```diff
@@ -55,11 +55,6 @@ function getNewAudioInputDevice(newDevices, localAudio) {
             availableAudioInputDevices[0].label !== '') {
             return availableAudioInputDevices[0].deviceId;
         }
-        // Otherwise we assume that we don't have any audio input devices
-        // to use and that's why disable microphone button on UI.
-        else {
-            APP.UI.setMicrophoneButtonEnabled(false);
-        }
```

Comment:
Just to confirm I understand this correctly. This is now replaced by updating the Audio and Video Icons in UI.onAvailableDevicesChanged. Is that correct?